plugin/evm/config.go

```diff
@@ -15,6 +15,7 @@ const (
 	defaultEthApiEnabled               = true
 	defaultNetApiEnabled               = true
 	defaultWeb3ApiEnabled              = true
+	defaultPreimagesEnabled            = true
```

Comment:
Can you remove this line? We should allow this to default to false.

Reply:
Done in 3bf563221